src/main/java/Domain/Exceptions/ItemOutOfStockExepction.java

```diff
@@ -0,0 +1,12 @@
+package Domain.Exceptions;
```

Comment:
maybe in external folder- cause also needed in the frontend and the backend?
if not- leave it here.

Reply:
are you talking about the Exceptions? its not external tho, its home made exceptions :)